src/components/MainMenu/SettingsMenu.js

```diff
@@ -1,56 +1,165 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { Icon, Button } from '../../ui/components';
+import Scroller from '../Scroller';
+import { Toggle, Text } from '../../ui/components/Fields';
 import MenuPanel from './MenuPanel';
 
-const SettingsMenu = ({
-  active,
-  onClickInactive,
-  handleAddMockNodes,
-  handleResetAppData,
-}) => (
-  <MenuPanel
-    active={active}
-    panel="settings"
-    onClickInactive={onClickInactive}
-  >
-    <Icon name="settings" />
-    <div className="main-menu-settings-menu">
-      <div className="main-menu-settings-menu__header">
-        <h1>Settings</h1>
-      </div>
-      <div className="main-menu-settings-menu__form">
-        <fieldset>
-          <legend>Developer Options</legend>
-          <p>During an active interview session, clicking this button will create mock nodes for
-           testing purposes.</p>
-          <Button
-            color="mustard"
-            onClick={handleAddMockNodes}
-          >
-            Add mock nodes
-          </Button>
-        </fieldset>
-        <fieldset>
-          <legend>Global App Settings</legend>
-          <p>Click the button below to reset all app data.</p>
-          <Button
-            color="mustard"
-            onClick={handleResetAppData}
-          >
-            Reset Network Canvas data
-          </Button>
-        </fieldset>
-      </div>
-    </div>
-  </MenuPanel>
-);
+class SettingsMenu extends PureComponent {
+  componentDidMount() {
+    this.setFontSize();
+  }
+
+  componentDidUpdate() {
+    this.setFontSize();
+  }
+
+  setFontSize = () => {
```

Comment:
I think that this and the component methods probably belong in `<App />`. At the moment SettingsMenu persists on the screen, but that might not always be the case.